libcodechecker/log/log_parser.py

```diff
@@ -435,21 +436,39 @@ def get_compiler_standard(compiler, language):
         return standard
 
     @staticmethod
-    def set(details):
+    def load_compiler_info(filename, compiler):
```

Comment:
Please review #1908 there is an extension which will get the compiler info for both languages which should change the `compiler_info.json` loading introduced here if I'm right.

Reply:
OK, I'll rebase on top of it and add the logic when it lands.

Reply:
I'll also need to update the recently added `renew_info_files.py` debug script.

Reply:
And the other debug scripts in #1941.